src/apps/favorites-list/FavoritesList.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react";
+import EmptyList from "../../components/empty-list/empty-list";
+import usePager from "../../components/result-pager/use-pager";
+import { useGetList } from "../../core/material-list-api/material-list";
+import { useText } from "../../core/utils/text";
+import { Pid } from "../../core/utils/types/ids";
+import SearchResultListItemAdapter from "./materials/SearchResultListItemAdapter";
+
+export interface FavoritesListProps {
+  pageSize: number;
+}
+
+const FavoritesList: React.FC<FavoritesListProps> = ({ pageSize }) => {
+  const t = useText();
+  const { data } = useGetList("default");
+  const [displayedMaterials, setDisplayedMaterials] = useState<Pid[]>([]);
+  const [materials, setMaterials] = useState<Pid[]>([]);
+  const { itemsShown, PagerComponent } = usePager(materials.length, pageSize);
+
+  useEffect(
+    () => setDisplayedMaterials([...materials].splice(0, itemsShown)),
+    [itemsShown, materials]
+  );
+
+  useEffect(() => {
+    if (data && data.collections) {
+      setMaterials(data.collections as Pid[]);
```

Comment:
I think we should avoid type assertions if possible.
I think we cannot avoid it in this case, because the data fetching hook is generated by Orval.
But at least we can try to define the type as close to the source as possible.
Is it possible to do so in the beginning of the component like [this](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/f6b21e6ebf7ccc0eab3e0ba03326b32d06a8660b/src/apps/search-result/search-result.tsx#L114-L118)?